lib/interfaces.js

```diff
@@ -79,20 +79,30 @@ class Interfaces extends Cache {
       return;
     }
     if (iface.plugin) {
-      const [library, name] = iface.plugin.split('/');
-      const lib = metarhia[library] || npm[library];
-      if (!lib || !lib.plugins) return;
-      const plugin = lib.plugins[name];
-      if (!plugin) return;
-      script = plugin(iface);
-      iface = script();
+      this.loadPlugin(interfaceName, iface);
+      return;
     }
     for (const name of Object.keys(iface)) {
       const proc = new Procedure(script, name, this.application);
       this.changeInterface(interfaceName, name, proc);
     }
   }
 
+  loadPlugin(interfaceName, iface) {
+    const [library, name] = iface.plugin.split('/');
+    const lib = metarhia[library] || npm[library];
+    if (!lib || !lib.plugins) return;
+    const pluginSrc = lib.plugins[name];
+    if (!pluginSrc) return;
+    const context = this.application.sandbox;
+    const { exports } = metavm.createScript(name, pluginSrc, { context });
+    const plugin = exports(iface);
+    for (const [name, script] of Object.entries(plugin)) {
+      const proc = new Procedure(script, name, this.application);
+      this.changeInterface(interfaceName, name, proc);
+    }
```

Comment:
Why do we need this code duplication with `.change` method ?

Reply:
Here is a diff from hooks. For hooks script is already done earlier and hooks loaded within the existing script. For plugins we need to create Procedures for each plugin within the plugin script, not the file from api folder